volttrontesting/fixtures/docker_wrapper.py

```diff
@@ -47,29 +51,44 @@ def create_container(image_name: str, ports: dict = None, env: dict = None, comm
 
         # Create docker client (Uses localhost as agent connection.
         client = docker.from_env()
-        if ":" in image_name:
-            client.images.pull(image_name)
-        else:
-            # So all tags aren't pulled. According to docs https://docker-py.readthedocs.io/en/stable/images.html.
-            client.images.pull(image_name + ":latest")
-        container = client.containers.run(image_name, ports=ports, environment=env, auto_remove=True, detach=True)
+
+        try:
+            repo = image_name
```

Comment:
consider changing repo to be full_docker_image  then comment that full_docker_image is image plus tag

but repo doesn't make sense in this context for the variable name.  At least to me

Reply:
Agree. I was thinking of what to name this variable and looked at Python docker's images.pull(...) method for inspiration which used "repository" for its parameter. Full_docker_image seems more descriptive in this context.